wkcuber/wkcuber/download.py

```diff
@@ -0,0 +1,130 @@
+"""This module takes care of downsampling WEBKNOSSOS datasets."""
+
+from pathlib import Path
+from typing import Annotated, List, Optional
+
+import typer
+from rich import print as rprint
+
+from webknossos import BoundingBox, Dataset, Mag, webknossos_context
+from webknossos.client._defaults import DEFAULT_WEBKNOSSOS_URL
+from wkcuber.utils import parse_bbox, parse_mag
+
+app = typer.Typer(invoke_without_command=True)
+
+
+@app.callback()
+def main(
+    *,
+    path: Annotated[
+        Path,
+        typer.Argument(
+            show_default=False,
+            help="Path to save your WEBKNOSSOS dataset.",
+        ),
+    ],
+    exist_ok: Annotated[
+        bool,
+        typer.Option(
+            help="Is it alright to overwrite existing data on download path.",
+        ),
+    ] = False,
+    full_url: Annotated[
+        Optional[str],
+        typer.Option(
+            rich_help_panel="Args for full URL download",
+            help="URL where your dataset is available.",
+        ),
+    ] = None,
+    dataset_name: Annotated[
+        Optional[str],
+        typer.Option(
+            rich_help_panel="Download with dataset name",
+            help="Name of your dataset on your WEBKNOSSOS instance.",
+        ),
+    ] = None,
+    organisation_id: Annotated[
+        Optional[str],
+        typer.Option(
+            rich_help_panel="Download with dataset name",
```

Comment:
All the `rich_help_panel` text here are identical which might be right. I haven't run the module yet. Might also be a copy & paste issue.

Reply:
Add link where to find it: https://webknossos.org/auth/token

Reply:
Yes, in this case the identical content is intended because `rich_help_panel` groups options of the same category together.